lib/provider/camunda/parts/implementation/Listener.js

```diff
@@ -55,7 +55,7 @@ module.exports = function(element, bpmnFactory, options, translate) {
 
       var event = (listener.get('event')) ? listener.get('event') : '<empty>';
 
-      var label = (event || '*') + ' : ' + (LISTENER_TYPE_LABEL[listenerType] || '');
+      var label = (event || '*') + ' : ' + LISTENER_TYPE_LABEL[listenerType] || '';
```

Comment:
This should remain the same, cf.
```javascript
label = (undefined || '*') + ' : ' + undefined || '';
"* : undefined"

label = (undefined || '*') + ' : ' + (undefined || '');
"* : "
```

```suggestion
      var label = (event || '*') + ' : ' + (LISTENER_TYPE_LABEL[listenerType] || '');	
```

Reply:
done